InAppUtils/InAppUtils.m

```diff
@@ -35,6 +35,10 @@ - (void)paymentQueue:(SKPaymentQueue *)queue
                 NSString *key = RCTKeyForInstance(transaction.payment.productIdentifier);
                 RCTResponseSenderBlock callback = _callbacks[key];
                 if (callback) {
+                    if (transaction.error.code == SKErrorPaymentCancelled) {
```

Comment:
Would really recommend refactoring out this into its own function like I recommended here: https://github.com/chirag04/react-native-in-app-utils/pull/45#issuecomment-315418161